packages/itwinui-css/src/style/global.scss

```diff
@@ -49,10 +49,6 @@ html {
   }
 }
 
-.iui-anchor {
```

Comment:
Previously it was in global css then we decided to move it to its own folder but then we would break users layouts as they will need to import anchor css. So with major release we can do that now.

Reply:
this seems to be okay then?

Reply:
@bentleyvk But wont it happen here, that its a breaking change for react users??

Reply:
I suggest leave it here like that and in react we import anchor css in `useTheme` together with global, so users dont get breaking changes. In v2 release, can get rid of that.
@mayank99 might want to be aware of this

Reply:
I don't think it makes sense to keep it global. If people are using this class, it's expected that something will break with v1.

Reply:
css - yes, but react.. we are not doing there a major release, so we cant say users will have breaking change, but its not react direct fault :?

Reply:
using a class = css, not react. They should be using our `<Anchor>` component if they want to be shielded from breaking css changes.

Reply:
But it will break with the react minor version release. class is implementation detail (as is our global css inclusion in react), but technically, stuff would break.